src/main/target/MATEKF722/target.h

```diff
@@ -73,6 +73,10 @@
 #define USE_BARO_MS5611
 #define USE_BARO_BMP085
 
+//*********** Magnetometer / Compass ************* 
+#define MAG 
```

Comment:
This has to be `USE_MAG`. Please test your proposed changes before you open a pull request.